GoogleSignIn/Sources/GIDSignInInternalOptions.h

```diff
@@ -79,6 +89,13 @@ NS_ASSUME_NONNULL_BEGIN
                                          scopes:(nullable NSArray *)scopes
                                      completion:(nullable GIDSignInCompletion)completion;
 
++ (instancetype)defaultOptionsWithConfiguration:(nullable GIDConfiguration *)configuration
```

Comment:
This shouldn't be available on `TARGET_OS_MACCATALYST`.

Reply:
Done.